mkdocs.yml

```diff
@@ -2,7 +2,11 @@ site_name: Zanata Server
 docs_dir: docs
 repo_name: GitHub
 repo_url: http://www.github.com/zanata/zanata-server
+site_favicon: images/favicon.ico
```

Comment:
It actually is (a copy of) that one. I'm not sure mkdocs can take a url for the favicon, but let me check. If not, what's your take on either 1. having a copy of the file here, or 2. Not using a Zanata favicon.